sql/core/src/test/scala/org/apache/spark/sql/UDFSuite.scala

```diff
@@ -904,25 +905,43 @@ class UDFSuite extends QueryTest with SharedSparkSession {
 
   test("SPARK-34663: using java.time.Period in UDF") {
```

Comment:
Please, add SPARK-35777 to the test title.

Reply:
DOne